### DISCUSSION TITLE
How can i get the original path or the raw data of a file that has been copied to my dokan drive?

### DISCUSSION THREAD

**Discussion by Thomas-Giaroli** [dokan-dev/dokan-dotnet#337] [Q&A] [answered]
Hey there, i want to make it so when you drag & drop (or simply copy) a file onto my Dokan drive it gets uploaded to my distant server. My only problem is that i am unable to obtain the data the files contains, i just don't know how to do it, so i've thought about getting the original path to the file that has been copied, so that i can check if it still exists there and get the data from there, but i don't know how to put this in place too. So here i am, once again asking for your support. Thanks already

**Comment by Olof-Lagerkvist** [comment, marked as answer]
You can't get original path name for a copy operation at file system level. All you get is the data to copy, in the form of several write requests.
